wgsl/index.bs

```diff
@@ -1671,8 +1762,8 @@ type_decl
   | VEC3 LESS_THAN type_decl GREATER_THAN
   | VEC4 LESS_THAN type_decl GREATER_THAN
   | POINTER LESS_THAN storage_class COMMA type_decl GREATER_THAN
-  | decoration_list* ARRAY LESS_THAN type_decl COMMA INT_LITERAL GREATER_THAN
-  | decoration_list* ARRAY LESS_THAN type_decl GREATER_THAN
+  | attribute_list* ARRAY LESS_THAN type_decl COMMA INT_LITERAL GREATER_THAN
```

Comment:
we could move this out into an `array_type_decl` (without the `attribute_list` stuff)